IPython/core/interactiveshell.py

```diff
@@ -479,6 +479,11 @@ def input_splitter(self):
         """
     ).tag(config=True)
 
+    warn_venv = Bool(
+        True,
+        help='Warn if running in a virtual environment with no IPython installed (so IPython from the global environment is used).'
```

Comment:
You need to fix the formatting otherwise the pipeline won't pass:

```suggestion
        help="Warn if running in a virtual environment with no IPython installed (so IPython from the global environment is used)."
```